pkg/kubecost/asset.go

```diff
@@ -2012,11 +2057,44 @@ func (n *Node) add(that *Node) {
 	n.Adjustment += that.Adjustment
 
 	if n.Overhead != nil && that.Overhead != nil {
-
 		n.Overhead.RamOverheadFraction = (n.Overhead.RamOverheadFraction*n.RAMCost + that.Overhead.RamOverheadFraction*that.RAMCost) / totalRAMCost
 		n.Overhead.CpuOverheadFraction = (n.Overhead.CpuOverheadFraction*n.CPUCost + that.Overhead.CpuOverheadFraction*that.CPUCost) / totalCPUCost
 		n.Overhead.OverheadCostFraction = ((n.Overhead.CpuOverheadFraction * n.CPUCost) + (n.Overhead.RamOverheadFraction * n.RAMCost)) / n.TotalCost()
 	}
+
+	// Check and correct values by interpolation.
+	// For more detail, see ELV-102 https://kubecost.atlassian.net/browse/ELV-102
+	if accumulatingUniqueNode {
```

Comment:
kind of minor, but would something like addingUniqueNode work here as a var name? Does add know/care if it is part of an accumulation vs something else?

Reply:
I struggled with this name. (Got me!) If the node is unique, and we see two of them, then by definition we are accumulating---the same node can only be added with itself _over time_, as it cannot run twice at the same time. But `addingUniqueNode` is an equivalently good name. Anybody else want to help name this variable with us? :wink: